CONTRIBUTING.md

```diff
@@ -2,12 +2,34 @@
 Contribute to the repository by opening a pull request.
 
 ## Local
-Develop code locally by cloning the source code and installing it.
+Develop code locally by cloning the source code, creating a development environment and installing it.
+
+1. Install [mamba](https://mamba.readthedocs.io/en/latest/installation.html), then launch the `Miniforge Prompt`.
```

Comment:
please update the mamba link, it no longer works

Reply:
[https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html](Link) updated